### ISSUE TITLE
Add support for disabling external and internal interface/route/etc during onboarding script

### ISSUE THREAD

**Issue by memes** [memes/terraform-google-f5-bigip#26]
The latest DO package supports declaring almost all external and internal interface configuation parameters, but cannot do so if the onboarding script has already defined an interface for the same IP address.

As a BIG-IP deployer I want to be able to define internal and external interface parameters in a *Declarative Onboarding JSON file*. To do this, I must be able to disable the automatic configuration of the external and internal configs.

Depends on #22 and #23

**Comment by memes** [comment]
Closing as wontfix; the solution for #23 makes this request obsolete.
